primitives/src/signatures/bls_over_bn254.rs

```diff
@@ -120,6 +125,93 @@ impl SignatureScheme for BLSOverBN254CurveSignatureScheme {
     }
 }
 
+impl AggregateableSignatureSchemes for BLSOverBN254CurveSignatureScheme {
+    /// Aggregate multiple signatures into a single signature
+    fn aggregate(
+        _pp: &Self::PublicParameter,
+        sigs: &[Self::Signature],
+    ) -> Result<Self::Signature, PrimitivesError> {
+        if sigs.is_empty() {
+            return Err(ParameterError("no signatures to aggregate".to_string()));
+        }
+        let mut agg_point = sigs[0].sigma;
+        for sig in sigs.iter().skip(1) {
+            agg_point += sig.sigma;
+        }
+        Ok(Self::Signature { sigma: agg_point })
+    }
+
+    /// Verify an aggregate signature w.r.t. a list of messages and public keys.
+    /// It is user's responsibility to ensure that the public keys are
+    /// validated.
+    fn aggregate_verify<M: AsRef<[Self::MessageUnit]>>(
```

Comment:
Do we need this for hotshot?

Reply:
No, but it's the standard API for an aggregate signature, so I also keep it here.